remove pluck

ycczkl opened the pull request:
_.pluck was removed since lodash v4.00. Since _.map(https://lodash.com/docs/4.17.0#map) provided the same function, we should just use _.map in example.

cht8687 commented:
LGTM, @stevemao thoughts?

stevemao commented:
The change doesn't seem to just simplify revert https://github.com/you-dont-need/You-Dont-Need-Lodash-Underscore/pull/64?

ycczkl commented:
Basically I removed pluck section and added one more example to map section.
Since pluck was deprecated in lodash v4.0, there's no need to still show it as an example

cht8687 commented:
oh, @ycczkl , I think underscore still have this method in 1.8.3....since our repo is 'lodash-and-underscore', we might need to keep this info, isn't it?

ycczkl commented:
@cht8687 Yes, underscore still keeps it but actually _.map has exact the same function/ syntax as _.pluck in both underscore and lodash. I think _.map is more general and will always be support in future.
 It's fine if you wants to keep it. If you wants to keep it I think we can add a comment which point lodash v4.0 is not support it anymore.

cht8687 commented:
@ycczkl  yes, add a comment sounds a great idea. Developers will know it's no longer exist in Lodash 4.0.

cht8687 commented:
@ycczkl thanks!
